### ISSUE TITLE
Teleport service restart behaviour - deletes DynamoDB entries

### ISSUE THREAD

**Issue by rikaardhosein** [gravitational/teleport#746]
Scenario:

1. teleport.yaml file exists containing details about a few trusted clusters.
2. teleport service is started and this information gets uploaded to dynamodb.
3. a new trusted cluster needs to be added  so an entry is added to the dynamodb table but the configuration file isn't changed and the service isn't restarted.
4. eventually the service is restarted, and the entry that exists in dynamodb but not in the configuration file is deleted.

Is (4) desired behaviour? If so, why?  Is this going to remain like this in the future?

## [edit]
Ev: before releasing 2.0 we need to either document `seed_config` or remove it entirely.

**Comment by kontsevoy** [comment]
@rikaardhosein DynamoDb is not officially supported (yet). It's in master branch and the upcoming 2.0 release. I am guessing you're using community-supplied plugin for 1.3.x series?

**Comment by kontsevoy** [comment]
Also, try adding `seed_config: true` to `teleport.yaml` like this:

```yaml
teleport:
    # use YAML configuration as a "seed" into the database and then respect
    # additional entries from the DB.
    seed_config: true
```

**Comment by brumfb** [comment]
We are using some of the upcoming dynamodb functionality, we've tried with 1.5.x and 2.x-alpha.3.  We will try the seed_config to see if it solves our issue.

**Comment by kontsevoy** [comment]
@brumfb @rikaardhosein I see. Mandatory disclaimer: the master is not meant for production/serious use. Dynamic configuration is still in flex (that's why `seed_config` is undocumented and very likely to be removed in the official 2.0 release).

**Comment by klizhentas** [comment]
this should be fixed by #752

**Comment by kontsevoy** [comment]
Dupe. See #795
